packages/gestalt/src/Row.js

```diff
@@ -28,7 +28,8 @@ type Props = {|
   alignContent?: AlignContent,
   alignItems?: AlignItems,
   alignSelf?: AlignSelf,
-  children?: Node,
+  // $FlowExpectedError[unclear-type]
+  children?: ChildrenArray<Element<any>>,
```

Comment:
@dangerismycat could we change these to `children?: ChildrenArray<Element<*>>`? That way we might be able to get rid of the `$FlowExpectedError`

Reply:
@dangerismycat actually, should this be `ChildrenArray<?Element<*>>` (note the `?`) so we also support this case:
```
ChildrenArray<?Element<*>>
```
What do you think?

Reply:
works for me!